tunnel_endpoint.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+AGENT_TUNNEL_ENDPOINT=${AGENT_TUNNEL_ENDPOINT:-http://localhost:4040}
+TAILS_TUNNEL_ENDPOINT=${TAILS_TUNNEL_ENDPOINT:-http://localhost:4040}
+
+WAIT_INTERVAL=${WAIT_INTERVAL:-3}
+WAIT_ATTEMPTS=${WAIT_ATTEMPTS:-10}
+
+retrieve_endpoint () {
```

Comment:
This is more of a function that makes sure the server is ready rather than retrieving the endpoint.